inc/namespace.php

```diff
@@ -671,13 +671,8 @@ function cross_site_sso_redirect( $url ) {
 		?>
 	</form>
 	<?php // @codingStandardsIgnoreEnd ?>
-
-	<script>
-		setTimeout( function () {
-			document.getElementById( 'sso_form' ).submit();
-		}, 100 );
-	</script>
-
+	<?php // Removes line break and spaces from the script below, so it could be hash properly to be excluded from the Browser CSP. ?>
```

Comment:
```suggestion
	<?php // Removes line breaks and spaces from the script below, so it can be hashed properly to be excluded from the Browser CSP. ?>
```